v2/emailpassword/common-customizations/signup-form/additional-field-props.mdx

```diff
@@ -0,0 +1,253 @@
+
+:::
+
+
+## Creating Custom Components
+
+Currently, your sign-up form contains only email and password fields.
+While you can add additional simple input fields, the form also supports the integration of more sophisticated input types.
+These enhanced components can be radio buttons, checkboxes, dropdown, sliders, etc allowing for a more comprehensive user registration experience.
+
+Let's see how you can extend the Sign-up form to fit your needs.
+
+<FrontendSDKTabs>
+<TabItem value="reactjs">
+
+```tsx
+import SuperTokens from "supertokens-auth-react";
+import ^{recipeNameCapitalLetters} from "supertokens-auth-react/recipe/^{codeImportRecipeName}"
+import Session from "supertokens-auth-react/recipe/session";
+
+SuperTokens.init({
+    appInfo: {
+        apiDomain: "...",
+        appName: "...",
+        websiteDomain: "..."
+    },
+    recipeList: [
+        ^{recipeNameCapitalLetters}.init({
+            signInAndUpFeature: {
+                // highlight-start
+                signUpForm: {
+                    formFields: [{
+                        id: "select-dropdown",
+                        label: "Select Dropdown",
+                        optional: true,
+                        inputComponent: ({ value, name, onChange }) => (
+                            <select value={value} name={name} onChange={(e) => onChange(e.target.value)}>
+                                <option value="" disabled hidden>
+                                    Select an option
+                                </option>
+                                <option value="option 1">Option 1</option>
+                                <option value="option 2">Option 2</option>
+                                <option value="option 3">Option 3</option>
+                            </select>
+                        ),
+                      }, {
+                        id: "terms",
+                        label: "",
+                        inputComponent: ({ name, onChange }) => (
+                            <div
+                                style={{
+                                    display: "flex",
+                                    alignItems: "center",
+                                    justifyContent: "left",
+                                }}>
+                                <input name={name} type="checkbox" onChange={(e) => onChange(e.target.checked.toString())}></input>
+                                <span style={{ marginLeft: 5 }}>I agree to the terms and conditions</span>
+                            </div>
+                        )
+                    }]
+                }
+                // highlight-end
+            }
+        }),
+        Session.init()
+    ]
+});
+```
+</TabItem>
+</FrontendSDKTabs>
+
+<img alt="Prebuilt form UI with custom components" src="/img/emailpassword/signup-with-custom-components.png" />
```

Comment:
- change this to not have the dangling * char
- The terms and conditions thing also shows at the bottom of the sign up button - remove that
- make the links for the tos in the checkbox clickable